lib/open_project/text_formatting.rb

```diff
@@ -44,7 +44,7 @@ module TextFormatting
 
     # Truncates and returns the string as a single line
     def truncate_single_line(string, *args)
-      truncate(string.to_s, *args).gsub(%r{[\r\n]+}m, ' ')
+      truncate(string.to_s, *args).gsub(%r{[\r\n]+}m, ' ').html_safe
```

Comment:
`truncate` marks the string as `html_safe` [already](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate). The extra call is unnecessary.